lib/bundler/cli.rb

```diff
@@ -151,6 +151,8 @@ def check
         Bundler.rubygems.security_policy_keys.join('|')
     method_option "without", :type => :array, :banner =>
       "Exclude gems that are part of the specified named group."
+    method_option "with", :type => :array, :banner =>
+      "Include gems that are part of the specified named group."
```

Comment:
Yeah, that makes sense. I read the rest of the diff after I made that comment. :)